continuum_mechanics/src/solver/AbstractNonlinearElasticitySolver.hpp

```diff
@@ -1832,6 +1822,15 @@ double AbstractNonlinearElasticitySolver<DIM>::TakeNewtonStep()
         WARN_ONCE_ONLY("Linear solve (within a mechanics solve) didn't converge, but this may not stop nonlinear solve converging")
         // LCOV_EXCL_STOP
     }
+    else
+    {
+        // Throw an exception if the solver failed for any reason other than DIVERGED_ITS.
```

Comment:
In that case I guess we shuld exclude it from coverage? @fcooper8472

Reply:
Coverage ought to be unchanged by this commit.